dashboard/test/controllers/level_sources_controller_test.rb

```diff
@@ -35,7 +35,7 @@ class LevelSourcesControllerTest < ActionController::TestCase
     assert_equal true, app_options[:white_background]
     assert_equal [], app_options[:callouts]
 
-    options = assigns(:level_view_options)
+    options = assigns(:level_view_options_map)[@level_source.level_id]
```

Comment:
While moving level view options into view_options_helper.rb, I changed the instance variable from `@level_view_options` to `@level_view_options_map`, which can contain a LevelViewOptions struct for each level.